ocp_install_env.sh

```diff
@@ -181,6 +181,29 @@ EOF
     fi
 }
 
+function capabilities_stanza() {
+    if [[ -n "$BASELINE_CAPABILITY_SET" ]]; then
+cat <<EOF
+capabilities:
+  baselineCapabilitySet: "$BASELINE_CAPABILITY_SET"
+EOF
+    fi
+
+    if [[ -n "$BASELINE_CAPABILITY_SET" ]] && [[ -n "$ADDITIONAL_CAPABILITIES" ]]; then
+cat << EOF
+  additionalEnabledCapabilities:
+EOF
+    for cap in ${ADDITIONAL_CAPABILITIES//,/ }; do
+cat << EOF
+  - ${cap}
+EOF
+    done
+    elif [[ -n "$ADDITIONAL_CAPABILITIES" ]] && [[ -z "$BASELINE_CAPABILITY_SET" ]]; then
+    echo "Additional capabilities is set to: $ADDITIONAL_CAPABILITIES, but no desired BASELINE_CAPABILITY_SET is set"
```

Comment:
nit: missing indentation

Reply:
Sure thing, fixed, added some whitespace to help visually see the conditional blocks